testutil/testnode/node_interaction_api.go

```diff
@@ -177,7 +177,7 @@ func (c *Context) FillBlock(squareSize int, accounts []string, broadcastMode str
 	// in order to get the square size that we want, we need to fill half the
 	// square minus a few for the tx (see the square estimation logic in
 	// app/estimate_square_size.go)
-	shareCount := (squareSize * squareSize / 2) - 1
+	shareCount := (squareSize * squareSize / 2) - 2
```

Comment:
[out of curiosity] why did the size of one transaction occupy change from 1 to 2 shares? Did this PR fix a bug?

Reply:
> Did this PR fix a bug?

Yes it added math.Ceil to this line because our uint64 conversion was rounding down the number meaning occasionally we estimated too small of a square: https://github.com/celestiaorg/celestia-app/blob/11374a6953db1f15454621c4c195db197138927f/app/estimate_square_size.go#L37

I was thinking of extracting it out into a separate PR but we're about to overwrite this method entirely with ADR020